pkg/antctl/raw/check/installation/command.go

```diff
@@ -46,17 +46,21 @@ func Command() *cobra.Command {
 	}
 	command.Flags().StringVarP(&o.antreaNamespace, "namespace", "n", o.antreaNamespace, "Configure Namespace in which Antrea is running")
 	command.Flags().StringVar(&o.runFilter, "run", o.runFilter, "Run only the tests that match the provided regex")
+	command.Flags().StringVar(&o.testImage, "test-image", o.testImage, "Container image override for the installation checker")
```

Comment:
maybe this should be `agnhost-image` instead of `test-image`?
I find it less confusing this way as the tests clearly require that the `/agnhost` command be included in this image

Reply:
Do you suggest to rename the parameter for `check cluster` too?

Reply:
I do have a preference for `antrea-agent-image`, but for that case we use the image more out of convenience. We could come up with an alternative Ubuntu-based image that would work just as well (as long as `bash` and `modprobe` are included. So we can keep the more generic name of `test-image` for `check cluster` if you prefer.

Reply:
Do you think we could publish an unified test image for both commands? I think agnhost will not be suffice in the future when we support validating wireguard, multicast etc. Then we would need to rename the parameter or provide test specific image parameters.

Reply:
Yes we could publish a unified image. It would be good if we could use the toolbox image, assuming we don't grow the size too much.

Reply:
I guess I should split the patch into two: a one-line change for bugfix, another for unifying the image and providing a parameter to override the image.